internal/rest/paymentcodes.go

```diff
@@ -0,0 +1,62 @@
+package rest
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+	golangtraining "github.com/ngavinsir/golangtraining"
```

Comment:
```suggestion
	"github.com/ngavinsir/golangtraining"
```
(nitpick) unnecessary rename